src/ResourceTypeRegistry.js

```diff
@@ -36,8 +56,16 @@ export default class ResourceTypeRegistry {
     if(description) {
       this._resourceTypes[type] = {};
 
+      // Merge description defaults into provided description
+      description = merge({}, this._descriptionDefaults, description);
+
+      // Make inverse lookup object for exceptions
+      description.behaviors.dasherizeOutput._inverseExceptions = invertObject(
```

Comment:
I'm pretty sure this statement isn't needed, since the `behaviors()` method (called below) now takes care of it.